lib/app/features/feed/views/components/feed_item/feed_item_footer/entity_reposts_button.dart

```diff
@@ -13,9 +13,14 @@ import 'package:ion/app/utils/num.dart';
 import 'package:ion/generated/assets.gen.dart';
 
 class EntityRepostsButton extends ConsumerWidget {
-  const EntityRepostsButton({required this.eventReference, super.key});
+  const EntityRepostsButton({
```

Comment:
The same. I would suggest renaming into ReelRepostsButton

Reply:
Same as well =)